spec/models/requests/aeon_url_spec.rb

```diff
@@ -67,14 +67,6 @@
       expect(subject).to include('Site=MARQ')
     end
   end
-  context 'when a thesis holding' do
```

Comment:
I think we should retain these specs, we should make sure that thesis holdings are routed to MUDD.